docs/src/content/docs/pt-br/index.mdx

```diff
@@ -26,11 +26,11 @@ import { CardGrid, Card } from '@astrojs/starlight/components';
 import AboutAstro from '~/components/about-astro.astro';
 
 <CardGrid stagger>
-	<Card title="Documentação de iluminar os olhos" icon="open-book">
+	<Card title="Documentação de encher os olhos" icon="open-book">
```

Comment:
I understand that "encher os olhos" is more common than "iluminar os olhos", here the intention was mostly to keep the thematic of shining/sparkling/light from "Documentation that de**lights**"

I would keep it the way it is despite being less expected.

```suggestion
	<Card title="Documentação de iluminar os olhos" icon="open-book">
```

Reply:
That's okay, @Yan-Thomas, thank you for your feedback! Just reverted the change for that sentence. Other suggestions for "Documentation that delights" would be:

- Documentação encantadora
- Documentação que encanta

"iluminar os olhos" works as well, it just feels slightly unnatural, but that's of course just my opinion.
Let me know if there are any other specific tasks in high-demand for this repo, I'll be happy to work on them.